microsoft-edge/webview2/how-to/webdriver.md

```diff
@@ -229,6 +229,12 @@ For more info about the `DebuggerAddress` property of the `EdgeOptions` object,
 
 Congratulations!  You've successfully automated a WebView2 project and driven WebView2 by using Selenium and Microsoft Edge WebDriver, by attaching Microsoft Edge WebDriver to a running WebView2 app.
 
+#### UWP projects and debugging
```

Comment:
```suggestion
### UWP projects and debugging
```
@DavidShoe - 
should the added section, "UWP projects and debugging", be promoted to the same level as the two headings that are above it?

Outline of heading levels per current PR:
h1: Automate and test WebView2 apps with Microsoft Edge WebDriver
- h2: Step 1: Download the WebView2API Sample
- h2: Step 2: Install Microsoft Edge WebDriver
- h2: Step 3: Add Selenium to the WebView2API Sample
- h2: Step 4: Choosing whether Microsoft Edge WebDriver should launch your app or attach to it
   - h3: Approach 1: Letting Microsoft Edge WebDriver launch your WebView2 app
   - h3: Approach 2: Attaching Microsoft Edge WebDriver to a running WebView2 app
- h2: Step 4a: Letting Microsoft Edge WebDriver launch your WebView2 app
- h2: Step 4b: Attaching Microsoft Edge WebDriver to a running WebView2 app
   - h3: Launching the WebView2 app with remote debugging enabled
   - h3: Attaching Microsoft Edge WebDriver to the launched WebView2 app
      - h4: UWP projects and debugging